packaging/windows/doinstaller_wix.cmd

```diff
@@ -127,7 +127,19 @@ IF %ERRORLEVEL% NEQ 0 (
   EXIT /B 1
 )
 
-if NOT DEFINED JSON_UPDATE_FILENAME set JSON_UPDATE_FILENAME=update-windows-prod-v2.json
+:: UpdateChannel is a Jenkins select parameter, one of: Smoke, Test, None
+echo UpdateChannel: %UpdateChannel%
+set JSON_UPDATE_FILENAME=update-windows-prod-v2.json
+IF %UpdateChannel% EQU Test (
+  set JSON_UPDATE_FILENAME=update-windows-prod-test-v2.json
+)
+IF %UpdateChannel% EQU Smoke (
+  set JSON_UPDATE_FILENAME=update-windows-prod-%KEYBASE_VERSION%.json
+)
+IF %UpdateChannel% EQU Smoke2 (
+  set JSON_UPDATE_FILENAME=update-windows-prod-%KEYBASE_VERSION%.json
+)
```

Comment:
@cjb did we talk about setting multiple of these keys at once for a single build? Or something like always setting a version-specific key, and then later overwriting the prod key with a different command? Does any of this matter?